### ISSUE TITLE
[Feat]: Add auto slide to carousel cards in the hero bar.

### ISSUE THREAD

**Issue by AKACHI-4** [ShivangM/workinn#11]
## What ?
I'd like to add effect so carousel cards in hero bar can be slide automatically after 5 seconds.

## Screenshot
![image](https://github.com/ShivangM/workinn/assets/99159580/7e327f47-4f1c-43b1-93bb-bb3d16fd5d5b)

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

<br>
<br>

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by ShivangM** [comment]
> ### Description
> ### Hey 👋, I am a Hacktoberfest'23 contributor
> ## What ?
> I'd like to add effect so carousel cards in hero bar can be slide automatically after 5 seconds.
> 
> ## Screenshot
> ![image](https://user-images.githubusercontent.com/99159580/277150053-7e327f47-4f1c-43b1-93bb-bb3d16fd5d5b.png)
> 
> ## Code of Conduct
> * [x]  I agree to follow this project's Code of Conduct
> 
> 
> 
> > **Kindly assign it to me under _HacktoberFest'23_**
> > **I will surely complete it ASAP**
> 
> ### Thank You

Good idea, I can assign it to you.

**Comment by AKACHI-4** [comment]
![image](https://github.com/ShivangM/workinn/assets/99159580/637481ed-0d2b-41f0-84b7-d49130928c5b)

Hey @ShivangM I think this error is coz there's no available guide to make a .env file in the docs, so kindly update there and guide me with all used env values in the project..

**Comment by ShivangM** [comment]
Hi @AKACHI-4 , thanks for pointing this out. I have added instructions to set up the Firebase project and how to add env. Please refer to the Readme file.

Also, I would have to add one more script for populating the categories collection. it will require some time. Please wait for some time for that. I will update you when the script is added.

You can setup the firebase project by that time.

**Comment by AKACHI-4** [comment]
Okay @ShivangM sure

**Comment by ShivangM** [comment]
> Okay @ShivangM sure

Hi, I have added a script to populate the database. You can follow the steps from Readme now. Just sync your repo. and once you are done with the firebase configuration you can run:

`yarn populate`

After that wait for database to populate then you can start server with:

`yarn dev`

**Comment by AKACHI-4** [comment]
hey bro @ShivangM thanks it's working now ... but i want to curious to know ... that why you r using firebase... 

actually in past months my twitter feed populated with the news that firebase is not performing well... 
i am in process and would love to go with appwrite a less code backend configuration tool... what's your say on this.. ?

**Comment by ShivangM** [comment]
> hey bro @ShivangM thanks it's working now ... but i want to curious to know ... that why you r using firebase...
> 
> actually in past months my twitter feed populated with the news that firebase is not performing well... i am in process and would love to go with appwrite a less code backend configuration tool... what's your say on this.. ?

Hey! Yes, I have used app write in one of my previous projects, it's a pretty good competition to Firebase. However, I selected Firebase over Appwrite just because of easy real-time synchronization. Personally, I feel Firebase is more secure in terms of safety and has been trusted for years. As a freelancing platform, this will include heavy real-time data synchronization as we have to manage chats between sellers and owners. We could write our own node js server but again maintaining it would be a burden for scalability.

These were the sole reasons I opted for Firebase.
